autogpt/prompts/generator.py

```diff
@@ -137,11 +140,26 @@ def generate_prompt_string(self) -> str:
         """
         return (
             f"Constraints:\n{self._generate_numbered_list(self.constraints)}\n\n"
-            "Commands:\n"
-            f"{self._generate_numbered_list(self.commands, item_type='command')}\n\n"
```

Comment:
The constraints refer to "the commands listed below", so this may have unintended effects.